screens/HomeFeed/Tabs/DMFeed/index.tsx

```diff
@@ -143,40 +151,49 @@ const DMFeed = ({navigation}: Props) => {
-            title:
-              user?.id !== item?.chatroom?.chatroom_with_user?.id
-                ? item?.chatroom?.chatroom_with_user?.name
-                : item?.chatroom?.member?.name!,
-            avatar:
-              user?.id !== item?.chatroom?.chatroom_with_user?.id
-                ? item?.chatroom?.chatroom_with_user?.image_url!
-                : item?.chatroom?.member?.image_url!,
-            lastMessage: item?.last_conversation?.answer!,
-            lastMessageUser: item?.last_conversation?.member?.name!,
-            time: item?.last_conversation_time!,
-            unreadCount: item?.unseen_conversation_count!,
-            pinned: false,
-            lastConversation: item?.last_conversation!,
-            chatroomID: item?.chatroom?.id!,
-            deletedBy: item?.last_conversation?.deleted_by,
-            isSecret: item?.chatroom?.is_secret,
-            chatroomType: item?.chatroom?.type,
-          };
-          return <HomeFeedItem {...homeFeedProps} navigation={navigation} />;
-        }}
-        onEndReached={handleLoadMore}
-        onEndReachedThreshold={0.1}
-        ListFooterComponent={renderFooter}
-        keyExtractor={(item: any) => item?.chatroom?.id.toString()}
-      />
+      {chatrooms?.length === 0 ? (
+        <View style={[styles.justifyCenter]}>
+          <Text style={styles.title}>It's nice to chat with someone</Text>
+          <Text style={styles.subTitle}>
+            Pick a person from FAB button and start your conversation
+          </Text>
+        </View>
+      ) : (
+        <FlatList
+          data={chatrooms}
+          renderItem={({item}: any) => {
+            const homeFeedProps = {
+              title:
+                user?.id !== item?.chatroom?.chatroom_with_user?.id
+                  ? item?.chatroom?.chatroom_with_user?.name
+                  : item?.chatroom?.member?.name!,
+              avatar:
+                user?.id !== item?.chatroom?.chatroom_with_user?.id
+                  ? item?.chatroom?.chatroom_with_user?.image_url!
+                  : item?.chatroom?.member?.image_url!,
+              lastMessage: item?.last_conversation?.answer!,
+              lastMessageUser: item?.last_conversation?.member?.name!,
+              time: item?.last_conversation_time!,
+              unreadCount: item?.unseen_conversation_count!,
+              pinned: false,
+              lastConversation: item?.last_conversation!,
+              chatroomID: item?.chatroom?.id!,
+              deletedBy: item?.last_conversation?.deleted_by,
+              isSecret: item?.chatroom?.is_secret,
+              chatroomType: item?.chatroom?.type,
+            };
+            return <HomeFeedItem {...homeFeedProps} navigation={navigation} />;
+          }}
+          onEndReached={handleLoadMore}
+          onEndReachedThreshold={0.1}
+          ListFooterComponent={renderFooter}
+          keyExtractor={(item: any) => item?.chatroom?.id.toString()}
```

Comment:
create variable to store `item?.chatroom` and `item?.chatroom?.chatroom_with_user`